res/controllers/Traktor-Kontrol-S2-MK3-hid-scripts.js

```diff
@@ -406,6 +437,7 @@ TraktorS2MK3.loadTrackHandler = function (field) {
         engine.setValue(field.group, "eject", field.value);
     } else {
         engine.setValue(field.group, "LoadSelectedTrack", field.value);
+        engine.setValue(field.group, "pfl", 1);
     }
```

Comment:
this is unrelated

Reply:
Yes, sorry, force of habit. I am always thinking of an option in the settings for enabling PFL on track loading.